### ISSUE TITLE
Wrong imports in created stubs

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stub-Generator#82]
### Describe the bug

For `TaggedTable`, the following imports are created:

```txt
from safeds.data.tabular.containers import Table
from safeds.data.tabular.containers.Column import Column
from safeds.data.tabular.containers.Row import Row
from safeds.data.tabular.containers.Table import Table
```

1. `Table` is imported twice.
2. The imports for `Column`, `Row`, and `Table` don't respect re-exports. All those classes are in the package `safeds.data.tabular.containers`.

### To Reproduce

Create stubs for `safe-ds`.

### Expected behavior

Imports should be correct.

**Comment by Masara** [comment]
@lars-reimann Is this still happening or can this be closed?

**Comment by lars-reimann** [comment]
It'll get closed once #112 is merged in. This can be done once the merge conflicts are resolved.
